bindings/java/src/main/java/org/ucx/jucx/examples/UcxReadBWBenchmarkSender.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (C) Mellanox Technologies Ltd. 2019. ALL RIGHTS RESERVED.
+ * See file LICENSE for terms.
+ */
+
+package org.ucx.jucx.examples;
+
+import org.ucx.jucx.UcxCallback;
+import org.ucx.jucx.UcxRequest;
+import org.ucx.jucx.ucp.*;
+
+import java.io.Closeable;
+import java.io.IOException;
+import java.net.InetSocketAddress;
+import java.nio.ByteBuffer;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Stack;
+
+public class UcxReadBWBenchmarkSender {
+
+    public static void main(String[] args) throws IOException {
+        Map<String, String> argsMap = new HashMap<>();
+        for (String arg: args) {
+            String[] parts = arg.split("=");
+            argsMap.put(parts[0], parts[1]);
+        }
+
+        String serverHost = argsMap.get("s");
+        int serverPort = Integer.parseInt(argsMap.get("p"));
+        long totalSize = Long.parseLong(argsMap.get("t"));
+
+        Stack<Closeable> resources = new Stack<>();
+
+        UcpContext context = new UcpContext(new UcpParams().requestWakeupFeature()
```

Comment:
there is some code duplication between client and server, maybe they can be in same class?